buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile

```diff
@@ -34,7 +34,6 @@ FROM ppc64le/centos:centos7
 ENV USER="jenkins"
 
 RUN yum -y install \
```

Comment:
I think we should have a `yum -y update` before the installs and a `yum clean all` after. Also applies to the subsequent yum commands.